python/paddle/fluid/contrib/slim/quantization/infer_quant_strategy.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import six
+import numpy as np
+import platform
+import os
+from .... import Executor
+from .... import io
+from .... import core
+from ....compiler import CompiledProgram
+from ....framework import IrGraph
+from ..core.strategy import Strategy
+import logging
+import sys
+
+__all__ = ['InferQuantStrategy']
+
+FORMAT = '%(asctime)s-%(levelname)s: %(message)s'
+logging.basicConfig(level=logging.INFO, format=FORMAT, stream=sys.stdout)
+logger = logging.getLogger(__name__)
+
+
+class InferQuantStrategy(Strategy):
+    """
+    The strategy for Post Training quantization strategy.
+    """
+
+    def __init__(self,
+                 start_epoch=0,
+                 end_epoch=0,
+                 int8_model_save_path=None,
+                 fp32_model_path=None):
+        """
+        Args:
+            start_epoch(int): The 'on_epoch_begin' function will be called in start_epoch. default: 0
+            end_epoch(int): The 'on_epoch_end' function will be called in end_epoch. default: 0
+            int8_model_save_path(str): The path to save model with int8_t weight.
+                            None means it doesn't save int8 model. defalut: None.

```

Comment:
> The path to save model with int8_t weight.
None (?) means it doesn't save int8 model

- it's confusing. do you meand fp32 programdesc + int8 weight?
- int8_t ->int8

Reply:
> * it's confusing. do you mean fp32 programdesc + int8 weight?
it will save int8 programdesc + fp32 weight, because mkldnn conv int8 kernel only support fp32 weight now.

Reply:
Got it. But the `int8_model_save_path` and its notes are too confusing. @bingyanghuang

Reply:
I will refine the notes